tests/pyincore/analyses/epfdamage/test_epfdamage.py

```diff
@@ -85,6 +85,25 @@ def run_with_base_class():
     # Run Analysis
     epf_dmg_tsu_re.run_analysis()
 
+    #############################################################################
+    # Galveston EPF damage
+    # Run epf damage
+    epf_dmg_eq_memphis = EpfDamage(client)
```

Comment:
Minor nitpick, the the variable has "memphis" in it, but the analysis is for galveston. Can we clean that up?